### DISCUSSION TITLE
Form submit with `enter` key

### DISCUSSION THREAD

**Discussion by chioio** [tusen-ai/naive-ui#3270] [Q&A] [answered]
What should I do with `enter` key to submit form. 🤯

**Comment by 07akioni** [comment]
Sorry for late response, a bit busy at work last week.

You can prevent default on input keydown.

**Comment by chioio** [comment, marked as answer]
I think so, but I solved this by `keydown.enter` listener on form. Thanks for you reply. 😁

**Reply by JenuelDev** [reply]
example?

**Reply by 07akioni** [reply]
@keydown.enter.prevent
